src/app.py

```diff
@@ -29,37 +29,21 @@ def say_hello_world():
     return {'result': "Flask says Hello World"}
 
 @app.route('/surgeo', methods=["GET", "POST"])
```

Comment:
These make sense for now, seems like we'll make another route in the future that goes something like: 
`@app.route('/prediction-comparison', methods=['POST']
x = dataAugmentation(data)
return run_surgeo(x) and zrp(x)`